quickstart.md

```diff
@@ -0,0 +1,154 @@
+# Quickstart Guide
+
+To start, create an [Algolia account](https://algolia.com) and index with your data. To learn more about getting up and running, view the [Algolia documentation](https://algolia.com/docs).

```

Comment:
If it's intended for people new to both Algolia and Alexa, we might smooth it a bit by linking to Algolia's Quick Start -- https://www.algolia.com/doc/guides/getting-started/quick-start/ -- instead of the doc root.